tests/func/test_checkout.py

```diff
@@ -603,3 +603,64 @@ def test_checkout_should_avoid_copy_for_existing_cache(
 
     assert "Removing before checkout" not in caplog.text
     assert get_inode(repo_dir.FOO) == foo_inode
+
+
+class TestCheckoutShouldCopyOnExistingLink(object):
```

Comment:
This one and the one down below look identical, don't they?

Reply:
refactored